runtime/autoload/provider/clipboard.vim

```diff
@@ -88,5 +81,11 @@ function! s:clipboard.set(lines, regtype, reg)
 endfunction
 
 function! provider#clipboard#Call(method, args)
-  return call(s:clipboard[a:method],a:args,s:clipboard)
+  if a:method == 'get'
+    let data = s:selections[a:args[0]].data  " empty or [lines, regtype]
+    let regtype = get(data, 1, 'v')
+    return [call(s:clipboard[a:method], a:args, s:clipboard), regtype]
```

Comment:
why split `s:clipboard.get` in half like this? It seem completely arbitarary to me.